doc/schemas/pm-context-schema.json

```diff
@@ -21,13 +22,16 @@
         "contextGroupName": {
           "type": "string"
         },
+        "comment": {
+          "type": "string"
+        },
```

Comment:
I added those more as notes to myself, but perhaps it makes sense to keep it. The issue is that there is inconsistency in how code meanings are defined (i.e., sometime code meaning is an abbreviation, such as IAUC, and sometimes it is fully expanded definition, such as "Regional Cerebral Blood Flow"). It is confusing, and we may want to include some clarification in the comment.